### PR TITLE
Fix UnicodeDecodeError when using blogofile build

### PR DESCRIPTION
**Pull request by CYBAI**
Fix issue same as [blogofile issue #152](https://github.com/EnigmaCurry/blogofile/issues/152) and use `pre_build` function to solve this error.

### CODE REVIEW

File: _config.py [hunk 1]
```diff
@@ -57,6 +57,12 @@
     "requirements.txt"
 ]
 
+
+def pre_build():
+    import locale
```
Comment (rickmak) [inline _config.py:62]: Put the import to the top of the files.

**Comment by rickmak** [comment]
I cannot reproduce the error, what is your runtime? I am on 2.7.4

**Comment by CYBAI** [comment]
I'm on python 2.7.10

**Comment by rickmak** [comment]
Got you. Because HK people like me will not set the locale to chinese. And the system locale is already working. Will merge.

**Comment by rickmak** [comment]
Merged via 26bbb2d

**Pull request closed without merging**